web-src/src/services/ExpressSDKService.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+export class ExpressSDKService {
+  CDN_URL = 'https://cc-embed.adobe.com/sdk/1p/v4/CCEverywhere.js';
```

Comment:
I don't have a pref for reading this from Config, but thought I'd mention it in case you do have a stronger pref :)

Reply:
Totally forgot about this CDN_URL. I will move this to .env